string handlerName
string checkerName
---
time stamp
